_docs/applications/mssql.md

```diff
@@ -1,36 +1,40 @@
 ---
 layout: guide
-title: StorageOS Docs - Microsoft SQL Server
+title: Microsoft SQL Server with StorageOS
 anchor: applications
 module: applications/mssql
 ---
 
 
 # ![image](/images/docs/explore/mssqllogo.png) Microsoft SQL Server 2016 with StorageOS
 
- Microsoft's latest SQL Server release represents a major update with availability for Linux platform support as well support to run as a container.
+ Microsoft's latest SQL Server release represents a major update with availability for Linux platform support and support to run as a container.
 
 ## Linux Support
 
-For Ubuntu, the Docker container requires Xenial 16.04.
+Microsoft state SQL Server for Linux runs with any supported Linux distribution.  To date we have only tested with Ubuntu 16.04 which we used for this document and Alpine Linux (which did not work).  Additional requirements include:
+* Minimum of 4 GB of disk space 
+* Minimum of 4 GB of RAM (has been tested to work with 3.25GB)
+
+Starting with SQL Server vNext CTP 1.4, the SQL Server command-line tools are included in the Docker image. If you attach to the image with an interactive command-prompt, you can run the tools locally.
 
 ## More Information
```

Comment:
Ah - you want to move this to the More information section *and* move the section to the bottom of the page.